File: sign/sign-interface/src/main/java/com/drzyl/sign/vo/OrgVO.java
package com.drzyl.sign.vo;

import java.io.Serializable;

import lombok.Data;

/**
 * 组织机构 SYS_ORG 展示类
 */
@Data
public class OrgVO implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String orgId; // 组织机构编号

  protected String orgName; // 组织机构名称

  protected String orgType; // 组织机构类型

  protected String idPath; // 编号路径

  protected String namePath; // 名称路径

  protected Integer weightOrder; // 排序权重

  protected String orgPid; // 上级组织机构编号

  protected Integer versions; // 版本号

  protected String createId; // 创建人账号

  protected String createName; // 创建人名称

  protected Long createTimes; // 创建时间

  protected String modifyId; // 修改人账号

  protected String modifyName; // 修改人名称

  protected Long modifyTimes; // 修改时间

}

File: vue-coalsale/src/view/com/drzyl/sign/org/OrgVO.js
export default class OrgVO {
  constructor () {
    this.orgId = null // 组织机构编号
    this.orgName = null // 组织机构名称
    this.orgType = null // 组织机构类型
    this.idPath = null // 编号路径
    this.namePath = null // 名称路径
    this.weightOrder = null // 排序权重
    this.orgPid = null // 上级组织机构编号
    this.versions = null // 版本号
    this.createId = null // 创建人账号
    this.createName = null // 创建人名称
    this.createTimes = null // 创建时间
    this.modifyId = null // 修改人账号
    this.modifyName = null // 修改人名称
    this.modifyTimes = null // 修改时间
  }
}
